src/db.js

```diff
@@ -0,0 +1,119 @@
+const Sequelize = require('sequelize');
+const path = require('path');
+const { ensureDir } = require('./utils');
+
+const defaultDBPath = path.join(process.env.HOME, '.chlu', 'marketplace', 'db.sqlite');
+class DB {
+
+    constructor(options = {}) {
+        this.storage = options.storage || defaultDBPath;
+        this.dialect = options.dialect || 'sqlite';
+        this.username = options.username || 'username';
+        this.password = options.username || 'password';
```

Comment:
Require a password. If not supplied, fail.
Maybe we need a config file to be present. 

I am happy to move both the above recommendations into a separate issue.

Reply:
Ok, there's also a bad typo, I need to fix this.

Opened issue about using a conf file: https://github.com/ChluNetwork/chlu-marketplace-js/issues/6